pillar/elastalert.sls

```diff
@@ -499,10 +499,10 @@ elastic_stack:
                 must:
                   - query_string:
                       default_field: message
-                      query: S3ResponseError AND async_export_to_git
+                      query: S3ResponseError
                 filter:
                   - term:
-                      fluentd_tag: edx.cms.stderr
+                      fluentd_tag: edx.*
```

Comment:
Does the wildcard work in a term block? I don't know the specifics of the query DSL so it may be fine but it's worth validating.

Reply:
Thanks for noticing that. The wildcard does _not_ work in this term filter. I wonder if a filter on the fluentd tag is really necessary in the first place.

Reply:
I'm doing some queries and observing that the filter does help. Confining the result to particular fluentd tags doesn't have to be done with a `filter` clause, so I'm looking into an alternate way.

Reply:
Commit f6ea9218 should do the trick. I've tested the query, as follows (where localhost is tunneled to the ES cluster):
```
curl --location --request GET 'localhost:9200/logstash-mitx*-production*/_search' \
--header 'Content-Type: application/json' \
--data-raw '{
	"query": {
		"bool": {
			"must": [
				{"query_string": {"query": "S3ResponseError", "default_field": "message"}},
				{"query_string": {"query": "edx.cms.* OR edx.lms.*", "default_field": "fluentd_tag"}}
			]
		}
	}
}'
```